packages/unmock-core/src/service/parser.ts

```diff
@@ -0,0 +1,34 @@
+import jsYaml from "js-yaml";
+import { IServiceDef, IServiceDefFile, IServiceParser } from "../interfaces";
+import { Service } from "./service";
+
+const PATTERN_FOR_KNOWN_FILENAMES = /^(?:index|spec|openapi)\.ya?ml$/;
```

Comment:
Could move this to `constants`.
I would also add case insensitivity flag (so `OpenApi.yaml` will also match, for example).
IMO could also make it verbose with `new RegExp` (or `XRegExp`).

Reply:
Good point regarding the case insensitivity. I don't personally see any reason any other class would access this so I wouldn't put it to `constants.ts`, I'll just move it to be static class member.